### ISSUE TITLE
Left / right click issues in some apps.

### ISSUE THREAD

**Issue by JustCryen** [MartinPL/Tray-Icons-Reloaded#79]
**Describe the bug**

Some applications don't respond to left / right click.
For example I can't restore or manage Origin through the tray. (at least the version installed through Steam)
Also, I've tried using Wire (messaging app) and left clicking on its tray icon launches Wireshark.

Another issue is when you double [left] click (or spam click, sometimes it takes more than two click at a time) it starts focusing on random programs, it can be a minimized program or sometimes a program from a different workspace.

It's not something that broke recently or anything like that, It never worked correctly.
Other than these issues, everything else worked fantastically. Thank you for an awesome extension!

**Desktop:**
 - Distro: Arch Linux
 - GNOME Shell: 1:41.4-1
 - Extension version: 21
 - Display server: X11 

 I read the compatibility note: [✓]

**Comment by golompiki** [comment]
Generally, wine programs are not displaying menus after clicking on their tray icons. Left click don't bring window either. Most launchers are not working (i tested Origin, BattleNet and Epic). Battlenet sometimes work, I don't know why. Trays are not working with and without toggle "Modify wine apps behaviour" in settings. 

Arch
WIne few newest relases, Proton multiple versions, Proton GE, Wine Soda
X11 and Wayland
I use this extension for years and it never worked with wine trays.

**Comment by JustCryen** [comment]
Fair enough… but the "Wire" example still shows an existing issue with Linux native programs.
It seems like it searches by name and is prone to error.
